README.md

```diff
@@ -7,26 +7,27 @@ Basic usage and installation is described below. [Click here for complete Sophos
 ## Usage
 
 ```shell
-$ factoryctl --help   
 Usage: factoryctl <command> [options]
 
 Commands:
   factoryctl run             Execute pipeline or job
   factoryctl rerun [run-id]  Reschedule pipeline run
-  factoryctl delete          Delete specified resource
   factoryctl list            List specified resources
   factoryctl get             Get specified resource
   factoryctl create          Create specified resource
+  factoryctl delete          Delete specified resource
```

Comment:
Really, the commands should be in alphabetic order. `delete` was moved to the end of the list, because it is the least likely to be used... maybe?